src/api/store/utils/storage.ts

```diff
@@ -0,0 +1,14 @@
+import { getFromStore, setInStore } from '@api/db';
+
+export const storage = {
+  getItem: (key: string) => {
+    return Promise.resolve(getFromStore(key));
+  },
+  setItem: (key: string, value: unknown) => {
+    return Promise.resolve(setInStore(key, value));
+  },
```

Comment:
This looks slightly cleaner if you make the functions async and remove the `Promise.resolve`.